extensions/flyway/runtime/src/main/java/io/quarkus/flyway/runtime/FlywayDataSourceRuntimeConfig.java

```diff
@@ -119,6 +119,13 @@ public static FlywayDataSourceRuntimeConfig defaultConfig() {
     @ConfigItem
     public boolean validateAtStart;
 
+    /**
+     * true to quit exit right after the operation.
+     *
+     */
+    @ConfigItem
+    public boolean runAndExit;
```

Comment:
Are users meant to set this? If not, we should probably add warning that this is considered an advanced setting.

Reply:
@geoand do we have an example?

Reply:
Not that I recall. Just add some strongly worded statement since this stuff ends up in generated docs pages so we need to make sure users should not casually use this